js-exercises/least-common-multiple/leastCommonMultiple.js

```diff
@@ -0,0 +1,9 @@
+function leastCommonMultiple(...args) {
+  const gcd = (a, b) => (a ? gcd(b % a, a) : b);
+
+  const lcm = (a, b) => (a * b) / gcd(a, b);
+
+  return args.reduce(lcm);
+}
+
```

Comment:
Very slick code! I took a while to get the recursive GCD bit, but you get the benefit of the doubt. This reminds me, I need to work on recursion! You could definitely work on the nomenclature though :)